tests/Microsoft.Bot.Builder.Azure.Tests/CosmosDbStorageTests.cs

```diff
@@ -407,6 +409,106 @@ public async Task WaterfallCosmos()
             }
         }
 
+        [TestMethod]
+        public async Task DeleteAsyncFromSingleCollection()
+        {
+            string DocumentId = "UtteranceLog-001";
+            string[] keys = { DocumentId };
+            var storage = new CosmosDbStorage(new CosmosDbStorageOptions()
```

Comment:
This `CosmosDbStorage` instantiation is repeated in the three new test methods. Can we create a `private` method just to avoid repeated code?

Reply:
Hi Gasper, that sounds good, I will be working on that.

Reply:
@GasparAcevedoZainSouthworks Updated.